src/prisma/schema.prisma

```diff
@@ -331,9 +325,8 @@ model ContractTransactionReceipts {
   createdAt DateTime @default(now())
   updatedAt DateTime @updatedAt
 
-  @@unique([chainId, transactionHash])
-  @@index([contractId, timestamp])
-  @@index([contractId, blockNumber])
+  @@id([chainId, blockNumber, transactionHash])
+  @@index([timestamp, chainId, contractAddress])
```

Comment:
What is the query pattern? 
I believe we always queried by `chainId` and `contractAddress` and sometimes with timestamp? If that's the case, `timestamp` should be at the right-most column?